supabase/migrations/20260826141124_sso_providers_block_client_provider_id.sql

```
@@ -0,0 +1,100 @@
+END;
+$$;
+
+ALTER FUNCTION public.enforce_sso_provider_client_update_guard()
+  OWNER TO postgres;
+REVOKE ALL ON FUNCTION public.enforce_sso_provider_client_update_guard()
+  FROM PUBLIC;
+GRANT EXECUTE ON FUNCTION public.enforce_sso_provider_client_update_guard()
+  TO service_role;
+
+COMMENT ON FUNCTION public.enforce_sso_provider_client_update_guard() IS
+  'BEFORE INSERT/UPDATE trigger on public.sso_providers (per row). Runs on every '
```

SQLFluff (4.3.0)

[error] Line is too long (82 > 80).

(LT05)